core/CMakeLists.txt

```
@@ -0,0 +1,40 @@
+cmake_minimum_required(VERSION 3.14)
+
+project(core LANGUAGES CXX)
+
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+set(CMAKE_AUTOUIC ON)
+set(CMAKE_AUTOMOC ON)
+set(CMAKE_AUTORCC ON)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+find_package(QT NAMES Qt6 Qt5 COMPONENTS Core Quick REQUIRED)
+find_package(Qt${QT_VERSION_MAJOR} COMPONENTS Core Quick REQUIRED)
+
+find_package(Qt5Widgets REQUIRED)
+
+file(GLOB SOURCE_FILES ${PROJECT_SOURCE_DIR}/*.cpp)
+file(GLOB HEADER_FILES ${PROJECT_SOURCE_DIR}/*.hpp)
+file(GLOB HEADER_FILES_H ${PROJECT_SOURCE_DIR}/*.h)
+file(GLOB RESOURCE_FILES ${PROJECT_SOURCE_DIR}/*.qrc)
+
+set(CORE_INCLUDE_DIR ${CMAKE_CURRENT_SOURCE_DIR} CACHE INTERNAL "")
+
+qt5_add_resources(PLUGIN_RESOURCES ${RESOURCE_FILES})
+set(PROJECT_SOURCES
+    ${SOURCE_FILES}
+    ${PLUGIN_RESOURCES}
+    ${HEADER_FILES}
+    ${HEADER_FILES_H})
```

Избавляемся от `qt5_add_resources`
```suggestion
set(PROJECT_SOURCES
    ${SOURCE_FILES}
    ${RESOURCE_FILES}
    ${HEADER_FILES}
    ${HEADER_FILES_H})
```